Project.toml

```diff
@@ -23,7 +23,7 @@ FourierTools = "0.2.1"
 IndexFunArrays = "0.2"
 Interpolations = "0.12, 0.13"
 QuadGK = "2.4.1"
-SpecialFunctions = "1, 1.1, 1.2"
+SpecialFunctions = "1, 1.1, 1.2, 2"
```

Comment:
Why there was `1, 1.1, 1.2`? That's _**equivalent**_ to just `1`

Reply:
Probably because I didn't know what I was doing :D
Or maybe compat helper merged this?

Reply:
CompatHelper wouldn't do that since these aren't different major versions. For the record https://github.com/JuliaPhysics/PhysicalOptics.jl/commit/9538ef191598f002d11629210aa50dd9d56bb3ee

Reply:
Also, why is SpecialFunctions both in the main deps and in the extras? That's useless

Reply:
Thanks for pointing that out.
I try to fix that once I have time re-designing the general implementation.

Reply:
Wait, doesn't SpecialFunctions need to be mentioned in extras when I need it for testing purpose as well?

Reply:
No.